xbmc/settings/dialogs/GUIDialogLibExportSettings.cpp

```diff
@@ -145,8 +145,8 @@ void CGUIDialogLibExportSettings::OnSettingAction(const std::shared_ptr<const CS
 
   const std::string &settingId = setting->GetId();
 
-  if (settingId == CSettings::SETTING_MUSICLIBRARY_EXPORT_FOLDER && 
-      !m_settings.IsToLibFolders() && !m_settings.IsArtistFoldersOnly())
+  if (settingId == CSettings::SETTING_MUSICLIBRARY_EXPORT_FOLDER && !m_settings.IsToLibFolders() &&
```

Comment:
Looks unrelated.

Reply:
clang-format?

Reply:
I know but it touches code that is unrelated to this PR (same below).

Reply:
It may have auto trim on save in my IDE. Will check it later.